config/vsc_hortense.py

```diff
@@ -51,13 +63,13 @@ def command(self, job):
                 {
                     'name': 'cpu_rome_256gb',
                     'scheduler': 'slurm',
-                    'prepare_cmds': [common_eessi_init()],
+                    'prepare_cmds': [prepare_eessi_init, common_eessi_init()],
                     'access': hortense_access + ['--partition=cpu_rome'],
                     'environs': ['default'],
                     'descr': 'CPU nodes (AMD Rome, 256GiB RAM)',
                     'max_jobs': 20,
-                    'launcher': 'mympirun',
-                    'modules': ['vsc-mympirun'],
+                    'launcher': launcher,
+                    'modules': [mpi_module, env_module % {'partition': "cpu_rome"}],
```

Comment:
I'm not sure if I also need this in the job environment. loading these in run_reframe.sh is probably all I need. Gonna test that on our tier2 because I have not seen this cause trouble on hortense.